src/Microsoft.HttpRepl/Microsoft.HttpRepl.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk;Microsoft.DotNet.GlobalTools.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <PackAsTool>true</PackAsTool>
+    <AssemblyName>dotnet-httprepl</AssemblyName>
```

Comment:
Is this shipping as a package to nuget.org, a bundled command in the CLI, or both?

Reply:
The plan right now is only to ship it on NuGet.org, I expect us to re-examine that decision at some point though.